styles/Vocab/Plone/accept.txt

```diff
@@ -1,10 +1,22 @@
+async
+api 
+backend
+css
+http 
+https 
+JavaScript
+LTS 
+npm
+nvm 
+NodeJS
```

Comment:
I just checked and the correct term is `Node.js`. It is also a registered trademark, so a variation is also acceptable. See https://nodejs.org/en/about

```suggestion
Node.js
Node.js®
```